cachi2/core/package_managers/gomod.py

```diff
@@ -1011,7 +1011,7 @@ def _get_go_work_path(app_dir: RootedPath) -> Optional[RootedPath]:
     go = Go()
     go_work_file = go(["env", "GOWORK"], {"cwd": app_dir}).rstrip()
 
-    if not go_work_file:
+    if not go_work_file or go_work_file == "off":
```

Comment:
Why we need disabling go_work_file?

Reply:
@lkolacek it's fixing a bug in our code base that could affect the check later down the road for this preliminary support.